libtiny_tui/src/tests/mod.rs

```diff
@@ -281,3 +284,161 @@ fn ctrl_w() {
 
     expect_screen(screen, &tui, 30, 3, Location::caller());
 }
+
+// Tests text field wrapping (text_field_wrap setting)
+#[test]
+fn test_text_field_wrap() {
+    // Screen should be wide enough to enable wrapping. See SCROLL_FALLBACK_WIDTH in text_field.rs
+    // and MessagingUI::draw where we use the magic number '5' for min. number of lines that should
+    // be visible in the text field to be able to enable wrapping. (TODO: get rid of the magic
+    // number)
```

Comment:
This comment is invalid now, would be good to update it. Namely `SCROLL_FALLBACK_WIDTH` seems to be gone, and I don't know when we hide/show the nick now. Would be good to document that in the relevant module.